package/upgrade-matrix.yaml

```diff
@@ -0,0 +1,9 @@
+versions:
+- name: v1.1.0
+  minUpgradableVersion: v1.0.3
+- name: v1.0.3
+  minUpgradableVersion: v1.0.2
+- name: v1.0.2
+  minUpgradableVersion: v1.0.1
+- name: v1.0.1
+  minUpgradableVersion: v1.0.0
```

Comment:
I recommend moving the matrix file to the `harvester/harvester` repo. It's more natural to update the file when we need to cut a release (e.g.,: https://github.com/harvester/harvester/pull/2649)

Other parts lgtm, nice work!

Reply:
My bad. I thought the repo is the installer.